### PR TITLE
Generate API for ParseUI

### PR DESCRIPTION
**Pull request by noobs2ninjas**

### CODE REVIEW

File: Scripts/jazzy.sh [hunk 1]
```diff
@@ -18,4 +18,20 @@ jazzy \
   --framework-root Parse \
   --output docs/api
 
+jazzy \
+--objc \
+--clean \
+--author "Parse Community" \
+--author_url http://parseplatform.org \
+--github_url https://github.com/parse-community/Parse-SDK-iOS-OSX \
+--root-url http://parseplatform.org/Parse-SDK-iOS-OSX/ParseUI/api \
```
Comment (TomWFox) [inline Scripts/jazzy.sh]: can we switch this to https?

**Comment by TomWFox** [comment]
The jazzy ci build passed but I did notice this in the logs...
> /Users/distiller/project/ParseUI/Other/ParseUI.h:22:9: fatal error: 'ParseUI/PFCollectionViewCell.h' file not found

https://circleci.com/gh/parse-community/Parse-SDK-iOS-OSX/2983?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: Scripts/jazzy.sh (hunk 1 above)
Comment (noobs2ninjas) [reply Scripts/jazzy.sh]: Sure!

**Comment by noobs2ninjas** [comment]
I noticed that. Was just about to message you. I think the primary issue is, unlike parse whos projects uses xcodes internal grouping system, ParseUI uses actual files for grouping. Gotta check jazzys documentation to see how to get around that if we can. If not I guess I could switch it over. Wouldn't take long.

**Comment by noobs2ninjas** [comment]
I was just about to write the same thing. 

> The jazzy ci build passed but I did notice this in the logs...
> 
> > /Users/distiller/project/ParseUI/Other/ParseUI.h:22:9: fatal error: 'ParseUI/PFCollectionViewCell.h' file not found
> 
> https://circleci.com/gh/parse-community/Parse-SDK-iOS-OSX/2983?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

I thought I knew what it was. I thought it was that parse doesnt have to have jazzy traverse physical files. All classes are in the base folder but thats not true.

**Comment by noobs2ninjas** [comment]
I have no clue why its not finding that file. Is it getting any of them or does it just quit on the first one. Is there a way to work out how to solve this? Know anyone who would know how this works @TomWFox ?

**Comment by noobs2ninjas** [comment]
@ShawnBaek not entirely sure how that happened. Also, it seems I removed this branch some I'm not entirely sure how to remove those changes now.

**Pull request closed without merging**